src/slim-select/index.ts

```diff
@@ -113,25 +116,19 @@ export default class SlimSelect {
       this.render()
     }
 
+    this.functionNamespace = {
+        documentClick: this.documentClick.bind(this),
+        windowScroll: debounce(this.windowScroll.bind(this))
+    }
+
+
```

Comment:
```suggestion
```

Reply:
@Knagis can you merge the PR with your changes ?